game/script-greetings.rpy

```diff
@@ -972,6 +972,79 @@ label greeting_love_plus_show_yourself:
 
     return
 
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="greeting_love_plus_amazing_scenery",
+            unlocked=True,
+            affinity_range=(jn_affinity.LOVE, None),
+            additional_properties={
+                "expression": "2cslbol"
+            }
```

Comment:
We would also need to make sure the player already saw the manga event, even though players on love tier probably seen it, it never hurts to be sure. If you already took that in count and I didn't see, just ignore this

Reply:
This is already handled, though I can see where the confusion might come from. The conditional uses the desk item's unlock state, which in turn is unlocked by seeing the topic (or having seen the topic already when migrating).